### ISSUE TITLE
Hide `add you to list` notifications

### ISSUE THREAD

**Issue by SamVerschueren** [sindresorhus/anatine#90]
I couldn't care less to be honest. Are there people really caring in what lists they are added? I'd like to open the discussion because sometimes when you tweet something with a hashtag like `#npm`, loads of people add you to their personal `npm` list. I don't have and don't want to know that to be honest.

Thoughts?

**Comment by paulmolluzzo** [comment]
Didn't you make issue #82? LOL.

Not like I live for Twitter lists, but hiding content is something we should be careful to do. Is it really that big of a deal that we need to hide them?

**Comment by SamVerschueren** [comment]
Yes I did, because the style was incorrect :). Afterwards, I noticed that I actually don't care about that. I think it's a little bit like in `refined-github` with the star mentions being hidden. If you are someone like @sindresorhus with 1 billion stars per day, it really clutters up the view and you don't want that. If you aren't that popular, it's perfectly possible you'd like to know interest from other people in your repositories and don't want that hidden.

I think those notifications clutter up the view fairly quickly if suddenly 8 people add you to one of their lists. The only thing you see at that moment in your notifications is `Person X add you to @x/some-list`. But I agree, we should be careful when hiding content.

Most ideally it's an option, but I know @sindresorhus doesn't like options 😆 .

**Comment by paulmolluzzo** [comment]
I hear you on all this. Twitter makes it so hard to add people to lists I'm surprised anyone is getting added. 😄 

I don't have a strong preference either way, but wanted to point out that some other people might.

**Comment by SamVerschueren** [comment]
Absolutely, discussion is open for anyone who wants to chime in :).

**Comment by sindresorhus** [comment]
Yeah, let's hide it, 99% of list adds are spam or random useless adds.

**Comment by necolas** [comment]
FWIW, I would guess no one wants to see these notifications in the official apps either, and that ML models would remove them for most if not all users. Hopefully we'll deal with this on our end soon
